docs/data-pipeline.md

```diff
@@ -0,0 +1,127 @@
+# MBBS data pipeline
+
+This document describes:
+
+* sources of MBBS data
+* how each data from each source is transformed and cleaned
+* how the data are combined into the final artifacts
+* how the artifacts are versioned and made available to users
+* conventions used in the `mbbs` package related to the above
+
+## Key definitions
+
+Route
+: An MBBS survey route.
+Each route has 20 stops.
+
+Stop
+: A location on a route where a surveyer counts birds for 3 minutes
+according to the MBBS protocol.
+Each stop is separated by approximately 0.5 miles.
+
+## Data sources
```

Comment:
@IJBG - can you fill in/briefly describe datasets that you've worked on e.g. stop-level, observer, stop-coordinates, etc.

* What does the dataset contain (1-2 sentences)?
* What format is it in and where do you find in the repo?
* How often is it updated (if ever)

Reply:
@IJBG - do you think you'll be able to get this this week? If not, no worries and I'll do a first pass -- I'm motivated to get started redoing our data pipeline.

Reply:
@bsaul Yep, about to push the first pass, this can likely be pared down later. Sorry for the delay, I've been out most of the last week and am still under the weather. 

Not sure if I made the stop-coordinates, but it ought to include the lat/lons for each stop as well. I've got that over in my mbbs-analysis repo and will open an issue to move it over and address tomorrow.

Reply:
> l Yep, about to push the first pass, this can likely be pared down later. Sorry for the delay, I've been out most of the last week and am still under the weather.

Cool. No worries at all.